docs/loading/S3_Load.md

```diff
@@ -0,0 +1,272 @@
+# Load data from Amazon S3
+
+StarRocks provides two options for loading data from S3:
+
+1. Asynchronous loading using Broker Load (`WITH BROKER`)
+2. Synchronous loading using the `FILES()` table function
+
+Small datasets are often loaded synchronously using the `FILES()` table function, and large datasets are often loaded asynchronously using Broker Load. The two methods have different advantages and are described below.
+
+## Using Broker Load
+
+An asynchronous Broker Load process handles making the connection to S3, pulling the data, and storing the data in StarRocks.
+
+### Advantages of Broker Load
+
+- Broker Load supports data transformation, UPSERT, and DELETE operations during loading.
+- Broker Load runs in the background and clients don't need to stay connected for the job to continue.
+- Broker Load is preferred for long running jobs, the default timeout is 4 hours.
+- In addition to Parquet and ORC file format, Broker Load supports CSV files.
+
+### Data flow
+
+![Workflow of Broker Load](../assets/broker_load_how-to-work_en.png)
+
+1. The user creates a load job
+2. The frontend (FE) creates a query plan and distributes the plan to the backend nodes (BE)
+3. The backend (BE) nodes pull the data from the source and load the data into StarRocks
+
+### Typical example
+
+Create a table, start a load process that pulls a Parquet file from S3, and verify the progress and success of the data loading.
+
+#### Create a table
+
+Create a database for your table:
+
+```SQL
+CREATE DATABASE IF NOT EXISTS project;
+USE project;
+```
+
+Create a table. This schema matches a sample dataset that you can load from a StarRocks S3 bucket.
+
+```SQL
+DROP TABLE IF EXISTS user_behavior;
+
+CREATE TABLE `user_behavior` (
+    `UserID` int(11),
+    `ItemID` int(11),
+    `CategoryID` int(11),
+    `BehaviorType` varchar(65533),
+    `Timestamp` datetime
+) ENGINE=OLAP 
+DUPLICATE KEY(`UserID`)
+DISTRIBUTED BY HASH(`UserID`) BUCKETS 1 
+PROPERTIES (
+    "replication_num" = "1"
+);
+```
+
+#### Gather connection details
+
+Loading data from S3 requires having the:
+
+- S3 endpoint
```

Comment:
For all the three authentication methods used to connect to AWS, one of them requires the S3 endpoint? See https://docs.starrocks.io/en-us/main/integrations/authenticate_to_aws_resources.

Reply:
When I wrote "S3 endpoint" I meant the S3 URI.  I will look in the S3 docs and see what AWS calls that. 

Yes, we should include that link at the end and mention that in the spot where we say that the dataset is available to anyone with an AWS key and secret

Reply:
The AWS docs use `s3://`, `bucket` and `object key (object name)` so that is what I will use instead of endpoint.  They only use `endpoint` when accessing via `http://` or `https://` and not `s3://`